centeredUi/README.md

```diff
@@ -1,78 +1,77 @@
 
-## Getting Started - Style Sheet Deployment 
+## Getting Started - Style Sheet Deployment
 
-1. Download the ```ThemeCenterBrand.css``` file to your ADFS server, wherever you host your style sheets.
-    Note: It is recommended that you minify your CSS for a production environment  
+1. Download the ```ThemeCenterBrand.css``` file to your AD FS server, wherever you host your style sheets.
 
-2. Create a custom web theme using the following command in PowerShell: 
+    Note: It is recommended that you minify your CSS for a production environment.
 
-    ```New-AdfsWebTheme –Name custom -SourceName default –StyleSheet @{path=”c:\style\ThemeCenterBrand.css”}```
+2. Create a custom web theme using the following command in PowerShell:
+
+    ```New-AD FSWebTheme –Name custom -SourceName default –StyleSheet @{path="c:\style\ThemeCenterBrand.css"}```
 
 3. Apply the new custom web theme using the following command in PowerShell:
 
-    ```Set-AdfsWebConfig -ActiveThemeName custom```
+    ```Set-AD FSWebConfig -ActiveThemeName custom```
+
+4. Update the logo and background image. For details and image size recommendations, see [this post](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/azure-ux-web-theme-in-ad-fs).
 
-4. Update the logo and background image. For details and image size recommendations, see [this post](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/azure-ux-web-theme-in-ad-fs)
+## Getting Started - JavaScript Deployment
 
-## Getting Started - JavaScript Deployment 
+1. Download the ```paginatedOnload.js``` file to your AD FS server, wherever you host your JavaScript.
 
-1. Download the ```paginatedOnload.js``` file to your ADFS server, wherever you host your JavaScript. 
-    
-    Note: It is *__highly__* recommended that you minify your ```paginatedOnload.js``` before including it in a production environment. There are many popular tools online 
-    for minifying JavaScript code. Two popular choices are [minifier.org](http://www.minifier.org/) and [JSCompress](https://jscompress.com/).
+    Note: It is *__highly__* recommended that you minify your ```paginatedOnload.js``` before including it in a production environment. There are many popular tools online for minifying JavaScript code. Two popular choices are [minifier.org](http://www.minifier.org/) and [JSCompress](https://jscompress.com/).
 
-2. Modify your existing custom web theme from the style sheet deployment to include the new JavaScript. In PowerShell: 
+2. Modify your existing custom web theme from the style sheet deployment to include the new JavaScript. In PowerShell:
 
-    ```Set-AdfsWebTheme –TargetName custom -AdditionalFileResource @{Uri=’/adfs/portal/script/onload.js’; path="c:\paginatedOnload.js"}```
+    ```Set-AD FSWebTheme –TargetName custom -AdditionalFileResource @{Uri="/AD FS/portal/script/onload.js"; path="c:\paginatedOnload.js"}```
```

Comment:
PowerShell cmdlet needs to be Set-AdfsWebTheme with no space in "Adfs". 

Also, the URI needs to be "/adfs" not "/AD FS"